src/SDKs/CognitiveServices/dataPlane/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.SpellCheck</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services SpellCheck API.</Description>
+    <VersionPrefix>1.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.SpellCheck</AssemblyName>
+    <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;SpellCheck API;REST HTTPclient;SpellCheckSDK;netcore451511</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the Cognitive Services SpellCheck SDK. Included with this release is support for SpellCheck API.</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
```

Comment:
@ashku-ms remove this as it is not required to add explicitly any files/folder to the project. They get added implicitly.